.github/SECURITY.md

```diff
@@ -63,54 +63,6 @@ not apply to the Airflow core package. There are also Airflow providers released
-  periods of lower activity, prolonged lack of activity and participation will result in removal
-  from the team, pending PMC decision (the decision on removal can be taken by LAZY CONSENSUS among
-  all the PMC members on private@airflow.apache.org mailing list).
-
-* They are not supposed to reveal the information about pending and unfixed security issues to anyone
-  (including their employers) unless specifically authorised by the security team members, specifically
-  if diagnosing and solving the issue might involve the need of external experts - for example security
-  experts that are available through Airflow stakeholders. The intent about involving 3rd parties has
-  to be discussed and agreed up at security@airflow.apache.org.
-
-* They have to have an [ICLA](https://www.apache.org/licenses/contributor-agreements.html) signed with
-  Apache Software Foundation.
-
-* The security team members might inform 3rd parties about fixes, for example in order to assess if the fix
-  is solving the problem or in order to assess its applicability to be applied by 3rd parties, as soon
-  as a PR solving the issue is opened in the public airflow repository.
-
-* In case of critical security issues, the members of the security team might iterate on a fix in a
-  private repository and only open the PR in the public repository once the fix is ready to be released,
-  with the intent of minimizing the time between the fix being available and the fix being released. In this
-  case the PR might be sent to review and comment to the PMC members on private list, in order to request
-  an expedited voting on the release. The voting for such release might be done on the
-  `private@airflow.apache.org` mailing list and should be made public at the `dev@apache.airflow.org`
-  mailing list as soon as the release is ready to be announced.
-
-* The security team members working on the fix might be mentioned as remediation developers in the CVE
-  including their job affiliation if they want to.
-
-* Community members acting as release managers are by default members of the security team and unless they
-  want to, they do not have to be involved in discussing and solving the issues. They are responsible for
-  releasing the CVE information (announcement and publishing to security indexes) as part of the
-  release process. This is facilitated by the security tool provided by the Apache Software Foundation.
-
-* Severity of the issue is determined based on the criteria described in the
-  [Severity Rating blog post](https://security.apache.org/blog/severityrating/)  by the Apache Software
-  Foundation Security team
+Security issues in Airflow are handled by the Airflow Security Team. Details about the Airflow Security Team and how members of it are chosen can be found in the [Contributing documentation](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#security-team).
```

Comment:
```suggestion
Security issues in Airflow are handled by the Airflow Security Team. Details about the Airflow Security Team and how members of it are chosen can be found in the [Contributing documentation](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst).
```
`security-teams` section does not exist do we mean URL of some other page?

Reply:
It _will_ exist though, it's being added in this PR :)

Reply:
ohh mb slipped :)